types/V1.mli

```
@@ -51,8 +51,8 @@ module type TIME = sig
   type +'a io
   (** The type for potentially blocking I/O operation *)
 
-  val sleep: float -> unit io
-  (** [sleep nsec] Block the current thread for. {b FIXME:} remove float. *)
+  val sleep: int64 -> unit io
+  (** [sleep n] Block the current thread for n nanoseconds. *)
```

> Behaviour for negative values is implementation-defined

Please no, raise `Invalid_argument`.